src/main/java/core/basesyntax/service/UserDataInvalidException.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+public class UserDataInvalidException extends RuntimeException {
+    private String message;
```

Comment:
The 'message' field is not necessary here, as the superclass RuntimeException already contains a detail message. Instead, you should pass the message to the superclass constructor.